src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/ThemeManager.cs

```diff
@@ -116,7 +116,10 @@ internal static void ApplySystemTheme(IEnumerable windows = null, bool forceUpda
                 }
                 
                 SetImmersiveDarkMode(window, !requestedUseLightMode);
```

Comment:
Can you check if the call made in SetImmersiveDarkMode is supported on Windows10 or not ?

Reply:
Yeah, the attribute ( DWMWA_USE_IMMERSIVE_DARK_MOD ) that we are using here for SetImmersiveDarkMode, is available in Win11 only : https://learn.microsoft.com/en-us/windows/win32/api/dwmapi/ne-dwmapi-dwmwindowattribute#constants

So, we should move SetImmersiveDarkMode call inside the Win11 check

Reply:
Addressed.